scripts/bert/dataset.py

```diff
@@ -14,25 +14,32 @@
 # limitations under the License.
 """BERT datasets."""
 
-__all__ = ['MRPCDataset', 'ClassificationTransform', 'BERTTransform']
+__all__ = [
+    'MRPCDataset', 'QQPDataset', 'ClassificationTransform',
+    'BERTTransform', 'QNLIDataset', 'RTEDataset', 'STSBDataset',
+    'COLADataset', 'MNLIDataset', 'WNLIDataset', 'SSTDataset'
+]
 
 import os
 import numpy as np
+from mxnet.metric import Accuracy, F1, MCC, PearsonCorrelation, CompositeEvalMetric
 try:
     from tokenizer import convert_to_unicode
 except ImportError:
     from .tokenizer import convert_to_unicode
 from gluonnlp.data import TSVDataset
 from gluonnlp.data.registry import register
 
+
```

Comment:
Please follow PEP-8 style guide and use 2 blank lines to separate top-level function and class definitions with two blank lines.  https://www.python.org/dev/peps/pep-0008/#blank-lines It seems you use 3 lines throughout this PR. You may find https://github.com/google/yapf/ helpful for auto-formatting the changes introduced by your PR

Reply:
Thanks for your guide. 👍